src/app/rundown-view/components/rundown-view/rundown-view.component.ts

```diff
@@ -17,14 +18,15 @@ import { NotificationService } from '../../../shared/services/notification.servi
 })
 export class RundownViewComponent implements OnInit, OnDestroy {
   public rundown?: Rundown
-  private connectionStatusSubscription: EventSubscription
-  private rundownSubscription?: EventSubscription
-  private readonly logger: Logger
   public keyBindings: KeyBinding[] = []
   public keystrokes: string[] = []
   public rundownWasRemoved: boolean
   public isLoadingRundown: boolean = true
   public shouldShowPopupNotifications: boolean
+  protected readonly RundownMode = RundownMode
```

Comment:
A minor thing that would be nice is also to have a newline between public and private variables :)

Reply:
Space added!